flask_app/FlaskApp.py

```diff
@@ -18,12 +18,22 @@ def __init__(self):
         self.azure_container_handler = AzureBlobContainerHandler()
 
     def run(self):
+        self.app.add_url_rule('/frame_path_by_index_and_videoid',
+                              view_func=self.get_frame_path_by_index_and_video_id,
+                              methods=['POST'])
         self.app.add_url_rule('/frames_path_by_videoid', view_func=self.get_frames_path_from_video_id, methods=['POST'])
         self.app.add_url_rule('/video_path_by_id', view_func=self.get_video_path_by_id, methods=['POST'])
         self.app.add_url_rule('/get_videos_paths', view_func=self.get_videos_paths, methods=['GET'])
         self.app.add_url_rule('/upload_video', view_func=self.upload_video, methods=['POST'])
```

Comment:
Please read again about the methods (Get, Post, Put and Delete and see what is good for each route)
Also please rewrite the url routes to something more dynamic.